Src/EB/AMReX_EB2_2D_C.cpp

```diff
@@ -20,13 +21,17 @@ void set_eb_data (const int i, const int j,
     constexpr Real tiny = 1.e-15;
 #endif
 
-    const Real axm = apx(i  ,j  ,0);
-    const Real axp = apx(i+1,j  ,0);
-    const Real aym = apy(i  ,j  ,0);
-    const Real ayp = apy(i  ,j+1,0);
-    const Real apnorm = std::hypot(axm-axp,aym-ayp) + 1.e-30_rt;
-    const Real nx = (axm-axp) * (1.0_rt/apnorm);
-    const Real ny = (aym-ayp) * (1.0_rt/apnorm);
+    const Real axm = apx(i  ,j  ,0)*dx[1];
+    const Real axp = apx(i+1,j  ,0)*dx[1];
+    const Real aym = apy(i  ,j  ,0)*dx[0];
+    const Real ayp = apy(i  ,j+1,0)*dx[0];
+    const Real daxp = (axm-axp);
+    const Real dayp = (aym-ayp);
+    const Real apnorm = std::hypot(daxp,dayp) + 1.e-30_rt;
+    const Real nx = daxp * (1.0/apnorm);
```

Comment:
Could you revert the changes here and many other places that removed `_rt`?

Reply:
Oh, yes. The merge introduced some troubles. I went through the changes and fixed.